src/lib/updates/utils/feed.js

```diff
@@ -0,0 +1,45 @@
+import { isString } from 'lodash'
```

Comment:
had to move those helpers to the separate file to break circular dependency issue